examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/main/paywalls/PaywallsScreen.kt

```diff
@@ -1,21 +1,121 @@
 package com.revenuecat.paywallstester.ui.screens.main.paywalls
 
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.revenuecat.paywallstester.SamplePaywalls
+import com.revenuecat.paywallstester.SamplePaywallsLoader
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.Offering
+import com.revenuecat.purchases.models.StoreTransaction
+import com.revenuecat.purchases.ui.revenuecatui.PaywallDialog
+import com.revenuecat.purchases.ui.revenuecatui.PaywallDialogOptions
+import com.revenuecat.purchases.ui.revenuecatui.PaywallViewListener
 
 @Composable
-fun PaywallsScreen() {
-    Box(
+fun PaywallsScreen(
+    samplePaywallsLoader: SamplePaywallsLoader = SamplePaywallsLoader(),
+) {
+    var displayPaywallDialogOffering by remember { mutableStateOf<Offering?>(null) }
+
+    LazyColumn {
+        items(SamplePaywalls.SampleTemplate.values()) { template ->
+            Column(modifier = Modifier.fillMaxWidth()) {
+                Text(
+                    text = template.displayableName,
+                    style = MaterialTheme.typography.labelMedium,
+                    modifier = Modifier.padding(8.dp),
+                )
+                ButtonWithEmoji(
+                    onClick = {
+                        val offering = samplePaywallsLoader.offeringForTemplate(template)
+                        displayPaywallDialogOffering = offering
+                    },
```

Comment:
It only supports full screen for now. When more modes are added this can probably be cleaned up